Makefile

```diff
@@ -191,6 +192,100 @@ pr-check:
 build-project:
 	CURRENT_ENV=mas mvn clean install -Pprod -Pno-docker -Dskip.yarn -Psql -Pmultitenancy -Dmaven.javadoc.skip=true --no-transfer-progress -DtrimStackTrace=false
 
+.PHONY: build-integration-tests-common ## Builds integration-tests-common
```

Comment:
What's up with all the test stuff?  Lots of new stuff in the Makefile.  Is this related?

Reply:
@SafarMirek ^^^

Reply:
I have needed to run integration tests locally and @riprasad suggested to add them into Makefile. I should probably create a new PR for it.

Reply:
Done. #1852

Reply:
Nice.